actions/utility.js

```diff
@@ -0,0 +1,28 @@
+import { camelToSnake } from '../utility';
+
+export const getUrl = (playerId, options, url) =>
+  `${url(playerId)}${options.reduce((previous, current) =>
+    current.values.reduce((total, value) => `${previous}${current.queryParam}=${value}&`, ''), '?')}`;
+
+export const addQueryString = options => Object.keys(options).map(key => ({
```

Comment:
consider using the querystring module to handle the URL manipulation?

https://www.npmjs.com/package/querystring